src/hotspot/share/opto/memnode.hpp

```diff
@@ -106,7 +106,7 @@ class MemNode : public Node {
   static Node *optimize_simple_memory_chain(Node *mchain, const TypeOopPtr *t_oop, Node *load, PhaseGVN *phase);
   static Node *optimize_memory_chain(Node *mchain, const TypePtr *t_adr, Node *load, PhaseGVN *phase);
   // This one should probably be a phase-specific function:
-  static bool all_controls_dominate(Node* dom, Node* sub);
+  static bool all_controls_dominate(Node* dom, Node* sub, bool *dead_code = nullptr);
```

Comment:
```suggestion
  static bool all_controls_dominate(Node* dom, Node* sub, bool* dead_code = nullptr);
```
We put the `*` next to the type. Putting it with the variable is old style - so you will see it but we try to change it whenever we touch a line.